### PR TITLE
change  unit name

### PR DESCRIPTION
**Pull request by beliaev-maksim**
would it make sense to change a unitname according to the warning per https://github.com/canonical/operator/pull/822 ?

### CODE REVIEW

**Review by sed-i — COMMENTED** [review]
Hi @beliaev-maksim, thanks for the PR!
I still see other additional tests fail on my end.
Could you please (temporarily) add this change to reqs txt so we can have CI reflect it?
```diff
-ops
+git+https://github.com/canonical/operator#egg=ops
```

**Comment by beliaev-maksim** [comment]
you mean integrations that are not part of tox?
image


I ran tests on my fork, looks OK except libs check, but I assume because it is a fork
https://github.com/beliaev-maksim/grafana-k8s-operator/pull/1

**Comment by sed-i** [comment]
I mean unit tests.
Can you please change `requirements.txt` so that `ops` is taken from gh instead of from pypi?
```diff
-ops
+git+https://github.com/canonical/operator#egg=ops
```

**Comment by beliaev-maksim** [comment]
@sed-i 
it is in there:
image

or do you mean something else?

**Comment by sed-i** [comment]
I mean unit tests.
Can you please change `requirements.txt` so that `ops` is taken from gh instead of from pypi?
```diff
-ops
+git+https://github.com/canonical/operator#egg=ops
```

**Comment by sed-i** [comment]
Ok great, missed it.

**Comment by beliaev-maksim** [comment]
@sed-i 
about "check libs". It looks like some issue in pipeline action itself.
even bumping it to stable via #119 get it failed.
credentials/secrets ?

File: tests/unit/test_charm.py [hunk 1]
```diff
@@ -310,7 +310,7 @@ def test_primary_sets_correct_peer_data(self, mock_unit_ip):
         self.harness.set_leader(True)
         self.harness.charm.on.config_changed.emit()
         rel = self.harness.model.get_relation("grafana")
-        self.harness.add_relation_unit(rel.id, "grafana/1")
+        self.harness.add_relation_unit(rel.id, "grafana-k8s/1")
```
Comment (rbarry82) [inline tests/unit/test_charm.py:313]: This will cause inconsistencies.

**Review by rbarry82 — CHANGES_REQUESTED** [review]

**Comment by simskij** [comment]
I'm happy to merge with check libs failing, if everything else is sorted out. I do, however, think Ryan is right and that we need to sort out [this](https://github.com/canonical/grafana-k8s-operator/pull/118#issuecomment-1238085615) first

**Review by rbarry82 — APPROVED** [review]

**Comment by rbarry82** [comment]
Raised on MM for ops to look at, and the follow-up which rationalizes things LGTM.

**Comment by simskij** [comment]
Thanks again @beliaev-maksim! 🎉

**Pull request merged**